Makefile

```diff
@@ -215,27 +203,17 @@ docker-push: ## Push docker image with the manager.
 
 ##@ Build Dependencies
 
-## Location to install dependencies to
-LOCALBIN ?= $(shell pwd)/bin
-$(LOCALBIN):
-	mkdir -p $(LOCALBIN)
+# Do not use OPERATOR_SDK variable not to overwrite the user provided path
+bin/operator-sdk:
+	mkdir -p bin
+	curl -sSLo bin/operator-sdk \
+		https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_$(shell go env GOOS)_$(shell go env GOARCH)
+	chmod +x bin/operator-sdk
 
+# Install to bin/operator-sdk unless already there or explicit OPERATOR_SDK provided
 .PHONY: operator-sdk
-OPERATOR_SDK ?= $(LOCALBIN)/operator-sdk
-operator-sdk: ## Download operator-sdk locally if necessary.
-ifeq (,$(wildcard $(OPERATOR_SDK)))
-ifeq (,$(shell which operator-sdk 2>/dev/null))
-	@{ \
-	set -e ;\
-	mkdir -p $(dir $(OPERATOR_SDK)) ;\
-	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
-	curl -sSLo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_$${OS}_$${ARCH} ;\
-	chmod +x $(OPERATOR_SDK) ;\
-	}
-else
-OPERATOR_SDK = $(shell which operator-sdk)
-endif
-endif
+operator-sdk: $(OPERATOR_SDK)
```

Comment:
This goal deps are satisfied when user provided existing installation through `OPERATOR_SDK`, or when they did not but the file exists. When neither is the case `bin/operator-sdk` goal kicks in.